src/layers/conv.jl

```diff
@@ -34,6 +35,10 @@ function conv_bn(kernelsize, inplanes, outplanes, activation = relu;
     bnplanes = outplanes
   end
 
+  if preact
+    activations = (conv = activation, bn = identity)
+  end
```

Comment:
are all 4 combinations of the `rev` and `preact` working as they are supposed to? just checking

Reply:
There was one edge case not being handled, I think - should be working now